seqr/management/commands/transfer_families_to_different_project.py

```diff
@@ -1,21 +1,28 @@
+from collections import defaultdict
+
 from django.core.management.base import BaseCommand
 
 from seqr.models import Project, Family, VariantTag, VariantTagType, Sample
 from seqr.utils.search.utils import backend_specific_call
+from seqr.views.utils.airflow_utils import trigger_airflow_delete_families
 
 import logging
-logger = logging.getLogger(__name__)
 
+from settings import SEQR_SLACK_LOADING_NOTIFICATION_CHANNEL
+
+logger = logging.getLogger(__name__)
 
 def _disable_search(families):
     search_samples = Sample.objects.filter(is_active=True, individual__family__in=families)
     if search_samples:
         updated_families = search_samples.values_list("individual__family__family_id", flat=True).distinct()
+        updated_family_dataset_types = list(search_samples.values_list('dataset_type', 'individual__family__family_id').distinct())
         family_summary = ", ".join(sorted(updated_families))
         num_updated = search_samples.update(is_active=False)
         logger.info(
             f'Disabled search for {num_updated} samples in the following {len(updated_families)} families: {family_summary}'
         )
+        return updated_family_dataset_types
```

Comment:
It makes more sense to call `trigger_delete_families_dags` directly here, rather than to return and than have another conditional block farther down